app/pastes/get-paste/getPasteInteractor.js

```diff
@@ -40,6 +42,8 @@ class GetPasteInteractor {
         if (paste.isPublic()) {
             paste.increaseTotalViewsCount();
             await this.pasteStatisticsService.increaseViews(paste.getId());
+            const text = await this.textStorage.getText(paste.getId());
+            paste.changeText(text);
```

Comment:
You do not need to attach it inside, since PasteEntity is just a metadata of an actual paste. 
reponseBuilder.build({paste, text})